### PR TITLE
9032 - Current display mode can be deselected in the toolbar/tablet menu

### PR DESCRIPTION
**Pull request by ElderOrb**
**Test plan:** 

Steps to reproduce:
1) Click on Menu in the toolbar or HMD tablet
2) Click Display
3) Click on the current display mode
4) Display mode is unselected (display is not affected)

After fix: 
Display modes should be mutually exclusive in both desktop & HMD tablet 'Display' menu-es.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4838/HighFidelity-Beta-PR11793-da49ee8bee498b9d1d9280afbc6ccb325d566d40.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4838/HighFidelity-Beta-PR11793-da49ee8bee498b9d1d9280afbc6ccb325d566d40.exe

**Comment by YuppyPlays** [comment]
Testing complete:

Steps to reproduce: **[Checked in 7360 Master, confirmed]**

Click on Menu in the toolbar or HMD tablet **Check**
Click Display **Check**
Click on the current display mode **Check**
Display mode is unselected (display is not affected) **Check**

After fix:
Display modes should be mutually exclusive in both desktop & HMD tablet 'Display' menu-es. **Check**

Solid **Pass**

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -7082,6 +7082,7 @@ DisplayPluginPointer Application::getActiveDisplayPlugin() const {
     return _displayPlugin;
 }
 
+static const char* exclusionGroupKey = "exclusionGroup";
```
Comment (druiz17) [inline interface/src/Application.cpp]: This goes against our coding standard. Const variable name must be in all caps with underscores in between the words. Look at the coding standard doc for examples. https://docs.highfidelity.com/build-guide/coding-standards

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/5082/HighFidelity-Beta-PR11793-c4aa93710817fbc7a7179b96e79393b9c2f8dfdd.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/5082/HighFidelity-Beta-PR11793-c4aa93710817fbc7a7179b96e79393b9c2f8dfdd.dmg

**Pull request merged**